lib/src/accordion-group/AccordionGroup.tsx

```diff
@@ -4,9 +4,8 @@ import { getMargin } from "../common/utils";
 import { spaces } from "../common/variables";
 import useTheme from "../useTheme";
 import AccordionGroupAccordion from "./AccordionGroupAccordion";
-import AccordionGroupPropsType, { AccordionGroupAccordionContextProps } from "./types";
-
-export const AccordionGroupAccordionContext = createContext<AccordionGroupAccordionContextProps | null>(null);
+import AccordionGroupPropsType from "./types";
+import { AccordionGroupAccordionContext } from "../layout/AccordionGroupContext";
```

Comment:
I think this case is still considered as a circular dependency.
Context imported by AccordionGroup and AccordionGroup.Accordion but AccordionGroup is also imported into AccordionGroup.Accordion.

Reply:
![image](https://github.com/dxc-technology/halstack-react/assets/33158831/ff539d5d-f5ee-4295-a6f3-83694b8b8298)
As far as I know this is not considered a circular dependency, we will have to discuss it with the rest of the team.